project-react/src/pages/Tablee.js

```diff
@@ -0,0 +1,62 @@
+import axios from "axios";
+import { useState, useEffect } from "react";
+import "./Tablee.css";
+const Tablee = () => {
+  const [posts, setPosts] = useState([]);
+  const apiEndPoint = "https://api.github.com/users/ariv797/repos";
+  useEffect(() => {
+    const getPosts = async () => {
+      const { data: res } = await axios.get(apiEndPoint);

```

Comment:
what if the api fails ? could u add api handling ? 
for example if error 4XX or 5XX shows pop up of "sorry something went wrong ..etc "

Reply:
When the api fails, an excuse message appears " Something Went Wrong! ".
I added it.